ci/vcpkg.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "pyogrio",
+    "version": "0.4.0",
+    "dependencies": [
+        {
+            "name": "gdal",
+            "default-features": false
+        }
+    ],
+    "builtin-baseline": "601d7f32059066ac229d4efbd9179790ccc57b0f",
+    "overrides": [
+        {
+            "name": "libgeotiff",
+            "version": "1.7.0"
```

Comment:
I would have expected that only the baseline would be sufficient? (assuming the baseline is from a commit where libgeotiff is still at 1.7.0?)

Reply:
Provided I did it correctly (I picked commit immediately prior to 1.7.0 being merged), I would have expected the same.  However, I found it went ahead and tried to install 1.7.1 anyway, without setting this override.

I don't know that it is necessarily a problem for us to explicitly list it as a pinned version though - if only for documentation purposes, since this was the root of the problem.

Reply:
> I don't know that it is necessarily a problem for us to explicitly list it as a pinned version though - if only for documentation purposes, since this was the root of the problem.

Indeed

Reply:
@jorisvandenbossche @brendan-ward I am finding vcpkg manifests difficult to use. Do you know if there is a vcpkg mode that will fail if no vcpkg.json is found? I suspect it's continuing without one in my case, but can't verify that.

Reply:
@sgillies I think you are missing a "manifests" feature flag. Assuming this is about https://github.com/rasterio/rasterio-wheels/pull/90, will comment over there